sandboxed_containers/sandboxed-containers-release-notes.adoc

```diff
@@ -93,7 +92,7 @@ $ oc scale deployments/machine-config-operator -n openshift-machine-config-opera
 $ oc scale deployments/machine-config-operator -n openshift-machine-config-operator --replicas=1
 ----
 
-. Revert the 'seLinuxOptions' strategy of the `sandboxed-containers-operator-scc` to its original value of 'MustRunAs' by running the following command:
+. Revert the `seLinuxOptions` strategy of the `sandboxed-containers-operator-scc` to its original value of 'MustRunAs' by running the following command:
```

Comment:
[RH supplementary guide](https://redhat-documentation.github.io/supplementary-style-guide/#code-examples) suggests that literal values for entry are monospace
```suggestion
. Revert the `seLinuxOptions` strategy of the `sandboxed-containers-operator-scc` to its original value of `MustRunAs` by running the following command:
```